src/components/MapView/Download/report.tsx

```diff
@@ -40,23 +40,26 @@ function Report({ classes, open, reportType, handleClose }: ReportProps) {
     ? moment(new Date(analysisResult?.date)).format('YYYY-MM-DD')
     : '';
 
-  const document = (
-    <ReportDoc
-      t={t}
-      exposureLegendDefinition={analysisResult?.legend ?? []}
-      theme={theme}
-      reportType={reportType}
-      tableName="Population Exposure"
-      tableShowTotal
-      eventName={
-        reportType === ReportType.Storm
-          ? `Storm Report (${eventDate})`
-          : `Flood Report (${eventDate})`
-      }
-      mapImage={mapImage}
-      tableData={analysisData}
-    />
-  );
+  const document =
+    mapImage !== null ? (
```

Comment:
Could this happen in `ReportDoc` instead at the image level? It might make things a bit cleaner. wdyt?
It's fine like that otherwise, thanks for fixing this